MC-3055: Show security provider config using hz-mc conf (#477)
hazelcast/management-center-docs: docs/modules/deploy-manage/pages/switching-security-provider.adoc

Then, you can configure another local security provider. For help setting one up, see xref:security-providers.adoc[]

> == Fetching the Security Provider details

Reviewer: ```suggestion
== Fetching the security provider details
```

Reviewer: We're applying the Google Style Guide so headings are now in sentence case

> You can fetch the configuration details of the security provider to validate if set correctly using the `hz-mc conf` tool with the `security get` task.

Reviewer: ```suggestion
You can fetch the configuration details of the security provider to validate if this is set correctly using the `hz-mc conf` tool with the `security get` task.
```

> The sensitive fields like `password` or `clientSecret` are masked in the output.

Reviewer: ```suggestion
Sensitive fields such as `password` or `clientSecret` are masked in the output.
```
